spec/services/idv/session_spec.rb

```diff
@@ -128,6 +127,8 @@
   end
 
   describe '#create_profile_from_applicant_with_password' do
+    let(:is_enhanced_ipp) { false }
+    let(:proofing_components) { { document_check: 'mock' } }
```

Comment:
suggestion (non-blocking): Extract ProofingComponents creation to a private method.

(failed to submit earlier, adding now)